packer/template.json

```diff
@@ -77,13 +77,14 @@
         ["modifyvm", "{{.Name}}", "--boot1", "dvd"],
         ["modifyvm", "{{.Name}}", "--boot2", "disk"],
         ["modifyvm", "{{.Name}}", "--chipset", "ich9"],
-        ["modifyvm", "{{.Name}}", "--cpuidset", "00000001", "000306a9", "00020800", "80000201", "178bfbff"],
         ["modifyvm", "{{.Name}}", "--firmware", "efi"],
         ["modifyvm", "{{.Name}}", "--hpet", "on"],
         ["modifyvm", "{{.Name}}", "--keyboard", "usb"],
         ["modifyvm", "{{.Name}}", "--memory", "2048"],
         ["modifyvm", "{{.Name}}", "--mouse", "usbtablet"],
-        ["modifyvm", "{{.Name}}", "--vram", "9"]
+        ["modifyvm", "{{.Name}}", "--usbehci", "on"],
```

Comment:
This `--usbehci` option seems to be the one that causes the `storageattach` error I just mentioned. I can set all other values in this diff and it boots, but I get the VBoxManage error with `--usbehci on`.